atomicfu-gradle-plugin/src/main/kotlin/kotlinx/atomicfu/plugin/gradle/AtomicFUGradlePlugin.kt

```diff
@@ -85,23 +105,34 @@ private fun Project.configureDependencies() {
             getAtomicfuDependencyNotation(Platform.JS, version)
         )
         dependencies.add(TEST_IMPLEMENTATION_CONFIGURATION, getAtomicfuDependencyNotation(Platform.JS, version))
-        addCompilerPluginDependency()
+        addJsCompilerPluginRuntimeDependency()
     }
     withPluginWhenEvaluatedDependencies("kotlin-multiplatform") { version ->
+        addJsCompilerPluginRuntimeDependency()
         configureMultiplatformPluginDependencies(version)
     }
 }
 
-private fun Project.configureTasks() {
-    val config = config
-    withPluginWhenEvaluated("kotlin") {
-        if (config.transformJvm) configureJvmTransformation()
+private fun Project.configureMultiplatformPluginDependencies(version: String) {
+    val multiplatformExtension = kotlinExtension as? KotlinMultiplatformExtension ?: error("Expected kotlin multiplatform extension")
+    val atomicfuDependency = "org.jetbrains.kotlinx:atomicfu:$version"
+    multiplatformExtension.sourceSets.getByName("commonMain").dependencies {
```

Comment:
Can there be a situation when there are no `commonMain` or `commonTest` source sets in the project?